static/lang/re-en.json

```diff
@@ -4073,6 +4073,7 @@
                 "AdvancedMulticlassFeat": "Select a class feat up to half your character level.",
                 "Alignment": "Select an alignment.",
                 "Benefit": "Select a benefit.",
+                "BosunsCommand": "Choose a command.",
```

Comment:
Did you consider naming this more generically, or moving it under NPCAbility?

Reply:
Hm - let's move this to NPCAbility. Makes more sense.